Redo fix on background image

jimmidier opened the pull request:
Undone work
fixes #302

- [x] Used named z-index utils instead of magic numbers
- [x] Used pseudo element as video mask instead of the parent
- [x] Provided background video and siblings with proper z-indexes
- [x] Cleaned up some other existing z-indexes
- [x] A bit code formatting

jimmidier committed:
Add z index tailwind plguin

jimmidier committed:
Fix background video issue using pheudo element and the new z index util

jimmidier committed:
Replace magic numbers of z index with util

wicksipedia committed:
💚 fix linting issues

wicksipedia reviewed (approved):
lgtm - after I fixed a few linting errors

wicksipedia committed:
Rollback to previous version of nearform/github-action-check-linked-issues

1.4.5 has a bug - see https://github.com/nearform/github-action-check-linked-issues/issues/250
